src/themes/default/components/core/blocks/Microcart/Product.vue

```diff
@@ -7,9 +7,21 @@
     </div>
     <div class="col-xs flex pl35 py15 start-xs between-sm details">
       <div>
-        <div class="serif h4 name">
+        <router-link
+          class="serif h4 name"
+          :to="localizedRoute({
+            name: product.type_id + '-product',
+            params: {
+              parentSku: product.parentSku ? product.parentSku : product.sku,
+              slug: product.slug,
+              childSku: product.sku
+            }
+          })"
+          data-testid="productLink"
+          @click.native="$emit('product-navigate')"
```

Comment:
you can safely use $store.commit('ui/setMicrocart', false) here and get rid of this event
```suggestion
          @click.native="$store.commit('ui/setMicrocart', false)"
```

Reply:
Thanks. I have updated the PR.